### PR TITLE
feat(hardware-testing): Adds simple script to show moving a plunger up/down

### PR DESCRIPTION
**Pull request by andySigler**
# Overview

# Changelog

# Review requests

# Risk assessment

### CODE REVIEW

**Review by thassyopinto — APPROVED** [review]

File: hardware-testing/hardware_testing/opentrons_api/helpers_ot3.py [hunk 1]
```diff
@@ -188,6 +190,71 @@ async def home_ot3(api: OT3API, axes: Optional[List[OT3Axis]] = None) -> None:
...
+    speed: Optional[float] = None,
+) -> None:
+    """Move OT3 plunger position in a relative direction."""
+    current_pos = await api.current_position(mount=mount)
+    await api._backend.set_active_current(
+        {OT3Axis.of_main_tool_actuator(mount): motor_current}  # type: ignore[dict-item]
+    )
+    plunger_pos = current_pos[OT3Axis.of_main_tool_actuator(mount)]  # type: ignore[index]
+    target_pos = {OT3Axis.of_main_tool_actuator(mount): plunger_pos + position}
+    await api._move(
```
Comment (Laura-Danielle) [inline hardware-testing/hardware_testing/opentrons_api/helpers_ot3.py:252]: Is there a particular reason you're choosing to not use `move_to` and `move_rel`?

File: hardware-testing/hardware_testing/scripts/test_pipette.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        hold_current=0.1,
+        run_current=1.4,
+    ),
+    OT3Axis.P_L: GantryLoadSettings(
+        max_speed=MAX_SPEED_PIP,
+        acceleration=40,
+        max_start_stop_speed=0,
+        max_change_dir_speed=0,
+        hold_current=0.1,
+        run_current=1.4,
```
Comment (Laura-Danielle) [inline hardware-testing/hardware_testing/scripts/test_pipette.py]: Shouldn't run current be 1.0 now?

**Review by Laura-Danielle — COMMENTED** [review]
Two questions, and then I can approve.

File: hardware-testing/hardware_testing/opentrons_api/helpers_ot3.py (hunk 1 above)
Comment (andySigler) [reply hardware-testing/hardware_testing/opentrons_api/helpers_ot3.py:252]: `api.move_to/move_rel` aren't setup to move the plunger. The only access the `OT3API` gives towards setting plunger positions is with `api.aspirate/dispense`. During testing, we will want to specify plunger positions in millimeters, as well as be able to completely avoid all the logic implemented in `OT3PipetteHandler`.

File: hardware-testing/hardware_testing/scripts/test_pipette.py (hunk 2 above)
Comment (andySigler) [reply hardware-testing/hardware_testing/scripts/test_pipette.py]: you're right fixing now!

File: hardware-testing/hardware_testing/opentrons_api/helpers_ot3.py (hunk 1 above)
Comment (sfoster1) [reply hardware-testing/hardware_testing/opentrons_api/helpers_ot3.py:252]: https://opentrons.atlassian.net/browse/RCORE-260

**Pull request merged**